input/pagecontent/reference.md

```diff
@@ -999,6 +1000,72 @@ Rules types that apply to Extensions are: [Assignment](#assignment-rules), [Bind
   * valueCode from BinaryBirthSexValueSet (required)
   ```
 
+{%include tu-div.html%}
+The keyword `Context` can be used to specify the [context](http://hl7.org/fhir/defining-extensions.html#context) of an Extension. The value should be a quoted string when specifying a `fhirpath` context. The value should be an unquoted path that includes the name, id, or URL of the resource that contains the element being set as the context when specifying an `element` or `extension` context. Multiple contexts can be specified by using a comma-separated list. Using the `Context` keyword instead of using rules to assign directly to the `context` list on the Extension is recommended.
+
+**Examples:**
+
+* Defining an extension with a `fhirpath` context
+
+  ```
+  Extension: MyExtension
+  Context: "(Condition | Observation).code"
+  ```
+
+* Defining an extension with an `element` context on a core FHIR resource
+
+  ```
+  Extension: MyExtension
+  Context: Patient.contact.telecom
+  ```
+
+* Defining an extension with an `element` context on a Profile defined in FSH
+
+  ```
+  Profile: MyPatient
+  Parent: Patient
+  // rules on the Profile omitted
+
+  Extension: MyExtension
+  Context: MyPatient.contact.telecom
+  ```
+
+* Defining an extension with an `element` context that references the element's containing resource by URL. Note that the path uses square brackets to refer to a slice.
+
+  ```
+  Extension: MyExtension
+  Context: http://hl7.org/fhir/us/core/StructureDefinition/us-core-careplan#category[AssessPlan].coding
+  ```
+
+* Defining an extension with an `extension` context
+
+  ```
+  Extension: MyExtension
+  Context: http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination
+  // The extension could also be referenced by id:
+  // Context: capabilitystatement-search-parameter-combination
+  ```
+
+* Defining an extension with an `extension` context, where the context is part of a complex extension
+
+  ```
+  Extension: MyExtension
+  Context: http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination#required
+  // For a deeply-nested complex extension, separate each included extension's URL with a . character:
+  // Context: http://example.org/StructureDefinition/some-complex-extension#required.use
```

Comment:
I thought we had decided that you always use a FSHy path at the end -- so `required` or `required.use` would not be valid. You'd have to do `extension[required]` or `extension[required].extension[use]`. While it's less typing to support the other way, I think reducing the available ways to do something is a simplification worth considering.

Reply:
Ah, I had forgotten about this! I'll change this in the docs and in my implementation.